rpcs3/util/fixed_typemap.hpp

```diff
@@ -14,7 +14,7 @@ namespace stx
 		struct destroy_info
 		{
 			void** object_pointer;
-			unsigned long long created;
+			size_t created;
```

Comment:
We use `std::size_t`, not just size_t, and it's declared in appropriate header file which isn't included in fixed_typemap.

Reply:
You seem to misunderstood my comment...
size_t doesn't even exist in C++, in C++ it's called std::size_t and declared in a bunch of C++ headers.
size_t is declared in C, not C++, headers.
You shouldn't turn std::size_t into size_t.
And this one can remain unsigned long long as an exceptional case.